infrastructure/gonogo/gonogo.cc

```diff
@@ -0,0 +1,36 @@
+#include "infrastructure/gonogo/gonogo.hh"
+
+#include <string>
+
+namespace jet {
+
+GoNoGo::GoNoGo() {
+    gonogomessage.bq_name = "";
+    gonogomessage.ready_ = false;
+    gonogomessage.status_message_ = "";
+    comms_factory_ = std::move(std::make_unique<jet::MqttCommsFactory>());
+    publisher_ = comms_factory_->make_publisher("GoNoGo");
```

Comment:
Not super important, but might be a little cleaner if you use an initializer list to create + set the comms factory and publisher members.